src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString.split(';').forEach((declaration) => {
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. Although `forEach` is an iteration method, in this context, it is being used to perform actions similar to a loop. Consider using `map`, `filter`, and `reduce` to transform the string into an object in a more functional style.